policy/policy.sample.json

```diff
@@ -0,0 +1,78 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1",
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants, auto generated for POST.
+            "AppliedTo": [ # can attach multiple tenants to which the policy can be applied to.
+                {
+                    "Tenant":"tenant1", #optional if TenantID below is provided 
+                    "TenantID": "3A6E467F-370B-43EA-BCEC-C28912EDD752", # optional but good to have to relate things as names are not unique
+                    "Segment": "Segment1", # optional
+                    "SegmentID": "01CA06A0-429B-4EE1-9185-F17D5CE4D0F2",
+                },
+                {
+                    "Tenant":"tenant1",
+                    "Segment": "Segment2",
+                },
+            ],
+            "Tags": [
+                {
+                    "Key": "Platform",
+                    "Value": "Openstack",
+                },
+                {
+                    "Key": "TenantID",
+                    "Value": "D46FD18B-A381-4297-A6D4-9DA594EE71C8",
+                },
+            ], # meta data attached to policies for various external enviorments like openstack/kybernetes
+            "Direction" : "Egress", # can be Egress or Ingress.
+            "Peers": [
+                {
+                    "CidrBlock": "0.0.0.0/0", # IP from L3 header
+                },
+            ],
+            "Rules": [
+                {
+                    "Protocol": "ICMP",
+                    "PortRangeStart": 4000,
+                    "PortRangeEnd": 4001,
+                    "IcmpTypeCode": [0, 8], # we look for this only if protocol above is 17, else ignore.
+                    "IsStateful": true,
+                },
+                {
+                    "Protocol": "UDP",
+                    "PortRangeStart": 8000,
+                    "PortRangeEnd": 8100,
+                    "IcmpTypeCode": [0, 8], # Ignore
+                    "IsStateful": true,
+                },
+            ],
+            "Description": "hello there, security policies are fun!",
+        },
+        {
+            "Name": "policy2",
+            "PolicyID": "CF2D2BE2-4553-4C28-BD02-140CF83617A2", # unique identifier across tenants, auto generated for POST.
+            "AppliedTo": [ # can attach multiple tenants to which the policy can be applied to.
+                {
+                    "Tenant":"tenant2",
+                    "Segment": "Segment1",
+                },
+            ],
+            "Tags": [], # meta data attached to policies for various external enviorments like openstack/kybernetes
+            "Direction" : "Ingress", # can be Egress or Ingress.
+            "Peers": [
+                {
+                    "CidrBlock": "0.0.0.0/0", # IP from L3 header
+                },
+            ],
+            "Rules": [{
+                "Protocol": "TCP",
+                "PortRangeStart": 80,
```

Comment:
Yes, that format is nicer, but you then have to throw in a string in the middle of it.
    [ 80, 90, "200-300", 1000]
If that's fine then I'm good with that. Had suggested the list to avoid that (really wanted to use a tuple, actually). But the string one is nice as well.